src/index.html

```diff
@@ -17,6 +17,77 @@
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="container">
+      <a
+        href="index.html"
+        class="logo"
+      >
+        <img
+          src="./images/logo.png"
+          alt="Logo"
+        />
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a
+              href="#"
+              class="is-activ nav__link"
```

Comment:
Don't use spaces in `<a>` tag's `href` property. Also, ensure that class names represent the meaning of the content. The class name 'is-activ' seems to be a typo and should be 'is-active' if it's meant to indicate the active state.

Reply:
Delete spaces in <a> tag's href property. Fixed the error in the class name 'is-active'